packages/mui-babel-macros/MuiError.macro.d.ts

```diff
@@ -1,3 +1,3 @@
 export default class MuiError {
-  constructor(message: string);
+  constructor(message: string, ...args: string[]);
```

Comment:
Come across this since my test is the first `.ts` that uses `MuiError` with multiple parameters.